rpc/README.md

```diff
@@ -2048,6 +2050,38 @@ http://localhost:8114
 }
 ```
 
+### `set_network_active`
+
+Disable/enable all p2p network activity
```

Comment:
`received` callback is still working, so not `all` network activity is disabled.